sources/packages/web/src/App.vue

```diff
@@ -1,6 +1,27 @@
 <template>
   <v-app>
-    <Toast />
+    <snack-bar :snackBarOptions="snackBarOptions" />
     <router-view></router-view>
   </v-app>
 </template>
+<script lang="ts">
+import SnackBar from "@/components/generic/SnackBar.vue";
+import { SnackBarOptions } from "./types";
+import { ref, onMounted } from "vue";
+import useEmitter from "./composables/useEmitter";
+
+export default {
+  components: { SnackBar },
+  setup() {
+    const emitter = useEmitter();
+    const snackBarOptions = ref({} as SnackBarOptions);
+
+    onMounted(() => {
+      emitter.on("snackBar", (snackBarOptionsObject: SnackBarOptions) => {
```

Comment:
Does it need to be defined inside the `onMounted`?

Reply:
mm...good question, in vue 2 usually, `.on` are added inside `onmounted` or `created`, and `.off` is added inside `beforeDestroy` or `onunmount`. let me test without onmounted

Reply:
this was a temp fix for this first PR